ComponentKit/Core/CKComponentInternal.h

```diff
@@ -65,7 +65,7 @@
 @property (nonatomic, weak) UIView *rootComponentMountedView;
 
 /** For internal use only; don't touch this. */
-@property (nonatomic, strong, readonly) id scopeFrameToken;
+@property (nonatomic, strong, readonly) NSNumber *scopeFrameToken;
```

Comment:
Hm I'm not a fan of exposing this implementation detail. Is there really a need to do this? Couldn't we just use isEqual:/hash on id<NSObject>?